src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateActionPlanTest.kt

```diff
@@ -153,13 +153,16 @@ class CreateActionPlanTest : IntegrationTestBase() {
   }
 
   @Test
-  @Transactional
```

Comment:
Got rid of the test that asserts we can create an action plan with one goal; and massively simplified the test that asserts we can create an action plan with multiple goals